### ISSUE TITLE
A warning during Inference

### ISSUE THREAD

**Issue by HunterKruger** [zhouj8553/Easy-Finetune-GLM#1]
The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.
Setting `pad_token_id` to `eos_token_id`:50007 for open-end generation.
